subprojects/packagefiles/theora/meson.build

```diff
@@ -0,0 +1,85 @@
+project(
+  'theora',
+  'c',
+  'cpp',
+  version: '1.1.1',
+  meson_version: '>= 0.59.0',
```

Comment:
This should really be 0.49.0. Compatibility with Debian oldstable is recommended though not required.

Most usages of new meson functionality seems to revolve around using feature options as booleans.

Reply:
yeah...that's exactly what's being done.

Reply:
I'm using the following new features:

```
theora| WARNING: Project specifies a minimum meson_version '>= 0.49.0' but uses features which were added in newer versions:
theora| * 0.50.0: {'install arg in configure_file', 'not_found_message arg in dependency'}
theora| * 0.56.0: {'meson.project_source_root'}
theora| * 0.59.0: {'feature_option.disable_auto_if()', 'feature_option.allowed()'}
```

Reply:
The first and second can be worked around easily.

The last one is interesting. I cloned this branch locally and saw most usages were of the form:

```
dependency(x: required: y.enabled())
```

The thing is, if y is disabled, the dependency will be set to not found, even if realistically found.

@eli-schwartz might have more input.